src/data/nav-items.yaml

```diff
@@ -27,16 +27,12 @@
   pages:
     - title: Get started
       path: /developing/get-started/
-    - title: Frameworks
+    - title: Frameworks (official)
       path: /developing/frameworks/react/
+    - title: Frameworks (community)
```

Comment:
Small suggestion

```suggestion
    - title: Frameworks
      path: /developing/frameworks/react/
    - title: Community frameworks
```

Reply:
Thanks for the suggestion! I made the changes across the rest of the subpages too.